### DISCUSSION TITLE
Android GitHub Webview - Markdown editor bug

### DISCUSSION THREAD

**Discussion by Morsmalleo** [community/community#38910] [Mobile]
### Select Topic Area

Bug

### Body

Hello, I'm not sure whether this would be the correct place to ask this but recently I've noticed that the editor that allows us to edit our markdown files such as `README.md` seems to be experiencing some bugs.

The problem is directly when editing files, I've experienced the following problems...
- Sometimes unable to add extra empty lines for spaces when clicking enter on the on screen keyboard, other times I have to double click enter twice, because if it's clicked once it automatically removes the inserted empty line

- Sometimes closes the on screen keyboard when tapping *backspace* to try and clear a set of words

- when highlighting it sometimes refuses to keep the selected text highlighted, unless the on screen keyboard is closed and reopened, other times for some reason it will sometimes remove the highlighted words without any interaction after highlighting

- When trying to tap on a certain point of the file being edited, say for example I want to go from `Line 5` of my `.md` file to `Line 10` to edit some text, it refuses to actually stay at the line you want to edit, the on screen keyboard needs to be closed and reopened to get around this

I'm not sure if this is happening on PC's, but for me when editing files on GitHub through the Chrome App these bugs arise, when about 3-4 days ago everything was fine.

Hopefully it's not just me and someone else has been experiencing this.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Morsmalleo! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by zhangzeyu007** [reply]
”@人生海海，总要允许有人错过你，才- - [ ] [ ] 能赶上最好的相遇——张皓宸

**Reply by zhangzeyu007** [reply]
- 哦们说~~~~

**Reply by zhangzeyu007** [reply]
- [ ] - - - - [ ] 个朋友。

**Reply by zhangzeyu007** [reply]
****

**Comment by auvrwhiite** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> Hello, I'm not sure whether this would be the correct place to ask this but recently I've noticed that the editor that allows us to edit our markdown files such as `README.md` seems to be experiencing some bugs.
> 
> The problem is directly when editing files, I've experienced the following problems...
> - Sometimes unable to add extra empty lines for spaces when clicking enter on the on screen keyboard, other times I have to double click enter twice, because if it's clicked once it automatically removes the inserted empty line
> 
> - Sometimes closes the on screen keyboard when tapping *backspace* to try and clear a set of words
> 
> - when highlighting it sometimes refuses to keep the selected text highlighted, unless the on screen keyboard is closed and reopened, other times for some reason it will sometimes remove the highlighted words without any interaction after highlighting
> 
> - When trying to tap on a certain point of the file being edited, say for example I want to go from `Line 5` of my `.md` file to `Line 10` to edit some text, it refuses to actually stay at the line you want to edit, the on screen keyboard needs to be closed and reopened to get around this
> 
> I'm not sure if this is happening on PC's, but for me when editing files on GitHub through the Chrome App these bugs arise, when about 3-4 days ago everything was fine.
> 
> Hopefully it's not just me and someone else has been experiencing this.

**Comment by Stephen3333** [comment]
Make sure you create a pom.xml file and load your fucking dependencies.

**Reply by Morsmalleo** [reply]
Watch your fucking mouth! Deadset!

**Comment by Stephen3333** [comment]
Damn
